### ISSUE TITLE
Add exports.default - for es6 modules compatibility

### ISSUE THREAD

**Issue by radubrehar** [hughsk/clamp#7]
First 2 lines would be:

``` jsx
exports.default = clamp
module.exports = clamp
```

So people can import it like `import clamp from 'clamp'`

**Comment by radubrehar** [comment]
Not mandatory for tools like `webpack`, since they know how to handle the `import` even if `module.exports` alone is used in the module, but would be a nice addition anyway :)

**Comment by mreinstein** [comment]
@hughsk would you accept a PR for this? I'd be happy to submit.

**Comment by mreinstein** [comment]
We should just switch to es modules since it's a standard now.

**Comment by rocktimsaikia** [comment]
Recently created a clamp lib with ES module support for use in some personal projects. Sharing it here just in case anyone finds it useful:
https://github.com/rocktimsaikia/clamp-v2

**Comment by mreinstein** [comment]
It would have been nice to work on getting `clamp` updated in npm instead of clogging the ecosystem with another dupe module.

**Comment by rocktimsaikia** [comment]
Yeah true but since it does not seem to be active anymore, just created an updated version with ESM and type support.
